Runtime/CoreLib/Collections/Generic/KeyValuePair.cs

```diff
@@ -9,7 +9,7 @@ namespace System.Collections.Generic {
 	[IncludeGenericArguments(false)]
 	[Serializable]
 	[Imported]
-	public sealed class KeyValuePair<TKey, TValue> {
+	public struct KeyValuePair<TKey, TValue> {
 		public KeyValuePair(TKey key, TValue value) {
 		} 
 
```

Comment:
Where and how? Not saying you're wrong, just wondering. Are you talking about a Visual Studio extension or an extension method?